chef/cookbooks/neutron/templates/default/linuxbridge_conf.ini.erb

```diff
@@ -8,7 +8,8 @@
 #
 # tenant_network_type = local
 # Example: tenant_network_type = vlan
-tenant_network_type = vlan
+# NOTE(rhafer): tenant_network_type is longer used for ml2. The option is now
```

Comment:
> s/longer/no longer/

Fixed in https://github.com/toabctl/barclamp-neutron/commit/fc213eb8071c32aefef5ed4e833db517db9c5f8f